docs/getting_started/creating_an_admin.rst

```diff
@@ -87,7 +92,8 @@ After this, create the schema for these entities:
 .. note::
 
     This article assumes you have basic knowledge of the Doctrine2 ORM and
-    you've set up a database correctly.
+    you've set up a database correctly. You can learn more on Doctrine entities
+    int the [Databases and the Doctrine ORM](https://symfony.com/doc/current/doctrine.html) chapter.
```

Comment:
btw this is not markdown, its rst, so the link syntax is invalid

Reply:
fixed in my last commit